### PR TITLE
update feed to use member

### PR DESCRIPTION
**Pull request by ann26**
continuing #1046 

JSON feed

```
{
"rss": {"version": "2.0", "channel": 
{
"lastBuildDate": "Fri, 18 Oct 2019 08:45:38 -0000", 
"link": "http://changelog.kartoza.com/en/qgis/", 
"description": "These are the members of QGIS project.", 
"item": [{
"description": "", 
"end_date": "05 March 2020", 
"title": "Spatial Networks, Inc.",
"member_level": "Medium", 
"member_country": "United States of America", 
"image_url": "http://changelog.kartoza.com/media/images/projects/ae367e8695ac064fa15a4e0ee37714f796f26255.png", 
"link": "http://changelog.kartoza.com/en/qgis/sponsorshipperiod/oenuqs/", 
"member_url": "https://spatialnetworks.com/", 
"guid": "http://changelog.kartoza.com/en/qgis/sponsorshipperiod/oenuqs/", 
"start_date": "05 March 2019"
},
```

### CODE REVIEW

**Comment by ann26** [comment]
hi @andreasneumann (cc: @timlinux )
Here's the PR for the JSON and rss feed, let me know if this suits your needs. I will deploy it to live if you are happy with this.

File: django_project/changes/feeds/sponsor.py [hunk 1]
```diff
@@ -247,7 +248,7 @@ def description(self, obj):
          :returns: Description of the RSS Feed.
          :rtype: str
          """
-        return 'These are the latest sponsors of %s project.' % obj.name
+        return 'These are the members of %s project.' % obj.name
```
Comment (timlinux) [inline django_project/changes/feeds/sponsor.py]: maybe 'sustaining members' rather

File: django_project/changes/feeds/sponsor.py [hunk 2]
```diff
@@ -322,7 +323,7 @@ def description(self, obj):
          :returns: Description of the RSS Feed.
          :rtype: str
          """
-        return 'These are the past sponsors of %s project.' % obj.name
+        return 'These are the past members of %s project.' % obj.name
```
Comment (timlinux) [inline django_project/changes/feeds/sponsor.py]: members -> sustaining members

File: django_project/changes/feeds/sponsor.py [hunk 3]
```diff
@@ -59,7 +59,7 @@ def description(self, obj):
          :returns: Description of the RSS Feed.
          :rtype: str
          """
-        return 'These are the latest sponsors of %s project.' % obj.name
+        return 'These are the members of %s project.' % obj.name
```
Comment (andreasneumann) [inline django_project/changes/feeds/sponsor.py]: Maybe 'These are the current members of %s project.'

**Comment by andreasneumann** [comment]
Thanks @ann26 - please take into account the comments from Tim and me and then merge.

Thanks!

**Comment by ann26** [comment]
done @timlinux and @andreasneumann, I will deploy this to staging and live

**Pull request merged**